extensions/wikia/VideosModule/Modules/VideosModuleBase.class.php

```diff
@@ -122,7 +122,6 @@ public function getVideos() {
 	protected function getCacheKey() {
 		$cacheKey = wfSharedMemcKey(
```

Comment:
@armonr, that's true, but both Staff Picks and Categories use wfSharedMemcKey and call this parent method. Since wfSharedMemcKey is needed more than the more specific wfMemcKey, I opted to leave it and add the dbname manually for Related Videos.